packages/flutter/test/material/date_picker_test.dart

```diff
@@ -774,3 +777,70 @@ void _tests() {
     expect(tester.getBottomLeft(find.text('OK')).dx, 800 - ltrOkRight);
   });
 }
+
+void _screenConfigTests() {
+
```

Comment:
Please insert a comment (probably several) like this here:
```
// Regression test for https://github.com/flutter/flutter/issues/21383
```

Or, if it makes more sense, in the specific `testWidgets` tests.

Reply:
Done.